### DISCUSSION TITLE
Sonoff http requests w/ Python Tkinter

### DISCUSSION THREAD

**Discussion by Nu2Java** [arendst/Tasmota#14482] [Q&A] [answered]
Hi All, I "really" am hoping someone out there has information on this.  I am using Tasmota on a Sonoff Basic. I have a DS18b20 sensor hooked up to it. I can see the temp in the main Tasmota screen (works great).  Now I am trying to somehow request that temp info in a python tkinter app. I am also controlling the Sonoff switch from my tkinter app which works great sending the http request.  But when I try anything on getting the temp sensor data, I just get back a "Response 200", which appears to just verify the communication.  Any help would be great.  Thanks

**Comment by sfromis** [comment, marked as answer]
`http://<sonoffbasic>/cm?cmnd=Status+10`

Still, MQTT is recommended for easy operation.
